Library/Homebrew/cmd/style.rb

```diff
@@ -8,18 +23,94 @@ def style
       ARGV.formulae.map(&:path)
     end
 
+    Homebrew.failed = check_style_and_print(target, :fix => ARGV.flag?("--fix"))
+  end
+
+  # Checks style for a list of files, printing simple Rubocop output.
+  # Returns true if violations were found, false otherwise.
+  def check_style_and_print(files, options = {})
+    check_style_impl(files, :print, options)
+  end
+
+  # Checks style for a list of files, returning results as a RubocopResults
+  # object parsed from its JSON output.
+  def check_style_json(files, options = {})
+    check_style_impl(files, :json, options)
+  end
+
+  def check_style_impl(files, output_type, options = {})
+    files = [files] unless files.is_a? Array
```

Comment:
`String` doesn't support `to_a`.

I just removed the conversion line: anything coming in is `concat`-able with arrays; the conversion guard isn't really needed with the current code.